vendor.conf

```diff
@@ -122,7 +122,7 @@ github.com/cyphar/filepath-securejoin v0.2.1
 gopkg.in/square/go-jose.v2 v2.1.3
 golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
 github.com/cri-o/ocicni 84aa158d2bacf95147b2b0a89615dd665630f440
-github.com/projectatomic/libpod master
+github.com/containers/libpod bf741b3ea30f9431775f03cca081758efcb780b1
```

Comment:
I would rather have this be v0.8.3, since it makes bumps easier to understand (and look up in [the change log][1]).  Once we eventually shift to Go 1.11 modules, we'll get both version number and cryptographic hashes, but until then I'm not concerned about libpod devs shifting a tag on us ;).

[1]: https://github.com/containers/libpod/blob/master/changelog.txt

Reply:
Sure, I can change that. It makes a lot of sense given libpod is versioned.

Reply:
done